packages/target-tauri/src-tauri/src/lib.rs

```diff
@@ -115,6 +115,9 @@ pub fn run() {
         builder = builder
             .plugin(
                 tauri_plugin_window_state::Builder::new()
+                    // Disabled for webxdc, because we generate a new UUID label
+                    // every time we create a webxdc window,
+                    // so it doesn't make sense to store it. TODO.
```

Comment:
```suggestion
                    // so it doesn't make sense to store it. TODO (https://github.com/deltachat/deltachat-desktop/issues/4468)
```

Reply:
it is mentioned/tracked in this issue: https://github.com/deltachat/deltachat-desktop/issues/4468